docs/getting-started/wallets.md

```diff
@@ -1,289 +1,102 @@
 
-```bash
-btcli wallet new_coldkey --wallet.name <my_coldkey>
-```
-
-For example, 
-```bash
-btcli wallet new_coldkey --wallet.name test-coldkey
-```
-You will see the following terminal output. The mnemonic is hidden for security reasons.
-
-```text
-IMPORTANT: Store this mnemonic in a secure (preferably offline place), as anyone who has possession of this mnemonic can use it to regenerate the key and access your tokens.
-The mnemonic to the new coldkey is:
-**** *** **** **** ***** **** *** **** **** **** ***** *****
-You can use the mnemonic to recreate the key in case it gets lost. The command to use to regenerate the key using this mnemonic is:
-btcli w regen_coldkey --mnemonic **** *** **** **** ***** **** *** **** **** **** ***** *****
-```
-
-:::tip Regenerating the coldkey
-Make a note of the above command option `regen_coldkey` showing how to regenerate your coldkey in case you lose it.
-:::
+**Purpose**: A coldkey is primarily for secure TAO storage and high-risk transactions, as described below (**Also see in the diagram in [Operational uses of keys](#operational-uses-of-keys))**:
 
-### Creating a hotkey using `btcli` 
-
-If you plan to validate or mine in a subnet, you must create both a coldkey and a hotkey.
-
-First, create a coldkey as described above in the [Creating a coldkey using `btcli`](#creating-a-coldkey-using-btcli). Then, provide this coldkey as a parameter to generate a hotkey. This will pair the hotkey with the coldkey. See below.
-
-Use the below command to generate the hotkey. Replace `<my_coldkey>` with the coldkey generated above, and `<my_hotkey>` with a name for your hotkey.
-
-```bash
-btcli wallet new_hotkey --wallet.name <my_coldkey> --wallet.hotkey <my_hotkey>
-```
-
-For example, 
-```bash
-btcli wallet new_hotkey --wallet.name test-coldkey --wallet.hotkey test-hotkey
-```
-
-You will see the terminal log like below. The mnemonic is hidden for security reasons.
-```text
-IMPORTANT: Store this mnemonic in a secure (preferably offline place), as anyone who has possession of this mnemonic can use it to regenerate the key and access your tokens.
-The mnemonic to the new hotkey is:
-**** *** **** **** ***** **** *** **** **** **** ***** *****
-You can use the mnemonic to recreate the key in case it gets lost. The command to use to regenerate the key using this mnemonic is:
-btcli w regen_hotkey --mnemonic **** *** **** **** ***** **** *** **** **** **** ***** *****
-```
-:::tip Regenerating the hotkey
-Make a note of the above command option `regen_hotkey` showing how to regenerate your hotkey in case you lose it.
-:::
+- Funds (TAO tokens) in your Bittensor wallet are held in its coldkey.
+- Delegating and undelegating your TAO tokens.
+- Creating a subnet and obtaining a `netuid` for the newly-created subnet. The `netuid` is associated with the coldkey only because all subnet owner operations require high security and thus use the coldkey, which is always encrypted.
```

Comment:
```suggestion
- Creating a subnet and obtaining a `netuid` for the newly-created subnet. The `netuid` is associated with the coldkey because subnet owner operations should be secure and conducted infrequently. 
```